### DISCUSSION TITLE
Using CDN instead of bundle

### DISCUSSION THREAD

**Discussion by SKFrozenCloud** [quasarframework/quasar#9901] [CLI - Capacitor / Cordova modes] [answered]
Gapi (Google API) serves its javascript with a CDN which I have put inside index.template.html.

Does Quasar download the files from all the CDNs used in the app once when building for e.g. ios/android?

If Quasar **does not** download the files from the CDNs when building the app, the app will **dynamically** download and load the javascript which is a security issue and also the app won't work if the mobile phone is offline.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@SKFrozenCloud 
> Does Quasar download the files from all the CDNs inside `index.template.html` once when building for e.g. ios/android.

No.
